.circleci/config.yml

```diff
@@ -92,13 +79,6 @@ jobs:
             echo -e "[pypi]" >> ~/.pypirc
             echo -e "username = $PYPI_USERNAME" >> ~/.pypirc
             echo -e "password = $PYPI_PASSWORD" >> ~/.pypirc
-      - run:
-          name: create release
```

Comment:
Are we no longer automatically creating a github release?  Is this something we will have to do manually from here out?

Reply:
I can't remember if that was even working, mostly just curious.

Reply:
The release part hasn't worked in a while (for me at least) - I have been manually creating the release in Github.

Reply:
https://github.com/reactiveops/reckoner/issues/30

Reply:
Thanks.  Couldn't remember